src/PhpSpec/Listener/MethodNotFoundListener.php

```diff
@@ -70,7 +70,8 @@ public function afterExample(ExampleEvent $event)
             return;
         }
 
-        $classname = get_class($exception->getSubject());
+        $subject = $exception->getSubject();
+        $classname = null === $subject ? get_class() : get_class($subject);
```

Comment:
getting the class of the MethodNotFoundListener does not make any sense here. We should rather handle the case of not having the subject in a different way